client/src/app/data-types.ts

```diff
@@ -233,8 +233,8 @@ export namespace Model {
     success: boolean;
     result: any;
 
-    static resourceName = "credential";
-    static extPath = "verify";
+    static resourceName = "v3/credential";
+    static extPath = "verify/";
```

Comment:
It doesn't look like we need to have the trailing `/` here, since we build the URL in the component rather than relying on the service.

Reply:
Yah this is a "legacy" change, it doesn't use the extPath at all for this component